src/shipit_code_coverage/tests/test_uploader.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: Maybe we should release all test files in this repository to the Public Domain, just like [all Mozilla test files](https://www.mozilla.org/en-US/MPL/license-policy/#Licensing_of_Mozilla_Code)?

To do that, you could add the following header comment:

    # Any copyright is dedicated to the Public Domain.
    # http://creativecommons.org/publicdomain/zero/1.0/